src/js/views/PeaksList/PeakDetails/usePeakDetails.js

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'react';
+
+const usePeakDetails = ({ id }) => {
+  const [peakDetails, setPeakDetails] = useState(null);
+
+  const getPeakDetails = () => {
+    // TODO - get data from backend
+
+    // fetch("http://localhost:8080/api/peaks/".concat(id))
+    // .then(response => {
+    //     console.log(response);
+    //     if (!response.ok){
+    //         throw Error("Error fetching peaks");
+    //     }
+    //     return response.json();
+    // })
+    // .then((data) => {
+    //     setPeakDetails(data);
+    // })
+
+    // example data from backend
+    const data =
```

Comment:
Skomplikowana ta struktura, w jakim celu trzymane są te linki wszystkie?

Reply:
Backend takie coś zwraca, bo jeżeli jakiś atrybut jest związany z inną tabelą to jest link do niego